src/core/Microsoft.PowerToys.Settings.UI.Lib/ImageResizerProperties.cs

```diff
@@ -0,0 +1,112 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace Microsoft.PowerToys.Settings.UI.Lib
+{
+    public class ImageResizerProperties
+    {
+        public ImageResizerProperties()
+        {
+            ImageresizerSelectedSizeIndex = new IntProperty(0);
+            ImageresizerShrinkOnly = new BoolProperty(false);
+            ImageresizerReplace = new BoolProperty(false);
+            ImageresizerIgnoreOrientation = new BoolProperty(true);
+            ImageresizerJpegQualityLevel = new IntProperty(90);
+            ImageresizerPngInterlaceOption = new IntProperty();
```

Comment:
Is there any way to use the resx files from ImageResizer for these strings, since they already have localized equivalents?

Reply:
we are already using the resource files for strings though. Which strings need to be in the rsx files? Thanks!

Reply:
I think all the setting names for the settings window are already there in the resx files in the ImageResizer UI project (like `Ignore Orientation`, etc). We could copy those over since I don't think we can directly reference them. It's also available in other languages as well, but I think we need not worry about that for now.

Reply:
I think ill add conversion to other languages on another PR. I have already added all the strings in the resource file.